core/query/sql.ml

```diff
@@ -274,7 +274,11 @@ class virtual printer =
             Format.pp_print_string (order_by_clause n)
       | Union (mult, qs, n) ->
         let pp_sep_union ppf () = Format.fprintf ppf "\nunion %a\n" pp_all mult in
-        let pp_value ppf x = Format.fprintf ppf "(%a)" pr_q x in
+        let pp_value ppf x =
+          match q with  (* parenthesize safely wrt SQL standard *)
+          | Select _ -> Format.fprintf ppf "%a" pr_q x
```

Comment:
I might be confused, but isn't this equivalent to `pr_q ppf x`?

Reply:
Sorry, yes, there was a silly typo there

Reply:
I was just adapting what was there already, happy to simplify this if I can